src/components/EditMenu.vue

```diff
@@ -101,7 +101,7 @@
       v-show="widgetMode === WidgetMode.RegularWidgets"
       class="flex flex-col items-center justify-between w-full overflow-y-clip h-[35%]"
     >
-      <p class="text-lg font-semibold select-none">Current widgets</p>
+      <p class="text-lg font-semibold select-none"><span class="mr-1 text-slate-400 mdi mdi-swap-vertical"></span>Current widgets</p>
```

Comment:
@ES-Alexander it's ready for me, but the linter is failling, because of all the HTML tags in the same line. Will just let you fit before we merge. For that cases, the linter auto-fix will not do a good job, so I attach a fix suggestion below.

<img width="1431" alt="image" src="https://github.com/bluerobotics/cockpit/assets/6551040/a78b0981-1c74-4274-834d-79c838eda9fa">

```html
<p class="text-lg font-semibold select-none">
  <span class="mr-1 text-slate-400 mdi mdi-swap-vertical" />
  Current widgets
</p>
```